### PR TITLE
Fix #14: Disable "save to paper" option when user is not authenticated

### LINKED ISSUE #14: Disable "save to paper" option when user is not authenticated
If auth token is not granted to app, we can't save text to paper, so disable context menu option until app is authorised, and enable it when token is received by app.

### PR DESCRIPTION
**Pull request by Insiyaa**
Please review...

Fixes #14 .

### CODE REVIEW

**Comment by Insiyaa** [comment]
This works fine but after authorization when the `updateState()` function is called, the `authToken` is null. On reloading the extension, it works fine...

File: js/paper.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 const clientId = "client id here";
 
+
```
Comment (AdityaHirapara) [inline js/paper.js]: No need for an extra line here

File: js/paper.js [hunk 2]
```diff
@@ -17,6 +18,33 @@ var contextMenuItem = {
 
 chrome.contextMenus.create(contextMenuItem);
 
+chrome.storage.sync.get(['token'], function(res) {
+  var authToken = res.token;
+  if(!authToken) {
```
Comment (AdityaHirapara) [inline js/paper.js]: space after if keyword

File: js/paper.js [hunk 3]
```diff
@@ -17,6 +18,33 @@ var contextMenuItem = {
...
+    chrome.browserAction.setBadgeText({text:'!'});
+    chrome.browserAction.setBadgeBackgroundColor({color:"red"})
+    chrome.browserAction.setTitle({title: 'Please Sign-In'});
+    chrome.contextMenus.update('sendText', {visible: false});
+  }
+});
+
+chrome.storage.onChanged.addListener(function(changes, areaName) {
+    chrome.storage.sync.get(['token'], function(res) {
```
Comment (AdityaHirapara) [inline js/paper.js]: two space indentation

File: js/paper.js [hunk 4]
```diff
@@ -17,6 +18,33 @@ var contextMenuItem = {
...
+    var authToken = res.token;
+    if(!authToken) {
```
Comment (AdityaHirapara) [inline js/paper.js]: space here, too.

**Comment by AdityaHirapara** [comment]
Nice work!:ok_hand:

**Pull request merged**